WordPress/Classes/ViewRelated/Gutenberg/EditorMediaUtility.swift

```diff
@@ -251,8 +202,10 @@ class EditorMediaUtility {
     }
 }
 
-private class EmptyImageDownloaderTask: ImageDownloaderTask {
+private struct MeediaUtilityTask: ImageDownloaderTask {
```

Comment:
`Media`, instead of `Meedia`?

Reply:
Fixed in https://github.com/wordpress-mobile/WordPress-iOS/pull/23975 (you already approved by the time I pushed).